.github/workflows/full.yml

```diff
@@ -35,6 +35,8 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v2
       - name: Run validation
+        # if it's not a PR we skip
+        if: github.github_base_ref != ''
```

Comment:
Use a more explicit way: `github.event_name == 'pull_request'` or something like that.

Reply:
Fixed.